### ISSUE TITLE
"Manage Access policy" for container doesn't save the updates

### ISSUE THREAD

**Issue by sgojgini85** [Microsoft/AzureStorageExplorer#104]
#### Storage Explorer Version: ####
1.0.0

#### Platform: ####
Windows

#### Architecture: ####
ia32

#### Build Number: ####
20180415.2

#### Commit: ####
5d603052

#### Regression From: ####
It used to work but don't remember the version when it was working

#### Steps to Reproduce: ####

1. right click on blob container under storage account
2. Click on "Manage Access Policies ... "
3. If there are no access policies present, create one and save to exit the dialog box
4. Go back to "Manage Access Policies ..."
5. Try to update the existing Access Policy created in previous step by changing the "Expiry Time" and Click Save
6. Go back to "Manage Access Policies ..." and you can see that update wasn't change

#### Expected Experience: ####
After step 5 in reproduction step, you should be able to go back and see the updated values

#### Actual Experience: ####
Currently the update isn't saved unless done through Azure Portal then it is saved properly.

**Comment by craxal** [comment]
Thanks, good catch! Fortunately, we already caught this issue and have a fix merged.

Duplicate of #50.
